examples/ecs-firelens/container-definitions-fargate.template.json

```diff
@@ -0,0 +1,47 @@
+[
+  {
+    "name": "example",
+    "image": "<account-id>.dkr.ecr.<region>.amazonaws.com/<image-name>:latest",
```

Comment:
can we add
```
"portMappings": [{ "containerPort": 8000, "protocol": "tcp" }],
```

SInce the App is a simple server which listen on 8000..

Reply:
oh how can that happen? 0.0.0.0:8000 is your local host isn't it? while this fargate instance must run on certain ip

Reply:
Yeah you are correct. That can happen only when the app is running on localhost as well. That was because once I ran docker on local via Intellij and it kept running for days. But all this was without specifying any port mappings.